src/aggregate/functions/string.c

```diff
@@ -299,6 +299,19 @@ static int stringfunc_split(ExprEval *ctx, RSValue *result, RSValue **argv, size
   return EXPR_EVAL_OK;
 }
 
+int func_is_null(ExprEval *ctx, RSValue *result, RSValue **argv, size_t argc, QueryError *err) {
+  VALIDATE_ARGS("isnull", 1, 1, err);
+
+  if (argv[0]->t == RSValue_Undef) {
```

Comment:
|| RSValue_Null?

Reply:
changed to RSValue_Null that was set at `evalProperty`